### PR TITLE
fix(editor url): Fix source lang in editor url

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-608

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-server/1248)

### CODE REVIEW

**Comment by huangp** [comment]
:lgtm: -(Awaiting review) +Reviewed +(On QA) 

---

Review status: 0 of 3 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1248#-:-KNEwPYKlNOttQXrktRP:7167892)*

**Comment by djansen-redhat** [comment]
@zanata-jenkins retest this please

**Comment by djansen-redhat** [comment]
@aeng any idea why it bumps the user back to the project page, with no tabs selected?

File: zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab-languages.xhtml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     #{msgs['jsf.Languages']}
   </h2>
 
-  <h:form id="settings-languages-form" styleClass="l--push-bottom-0">
```
Comment (djansen-redhat) [inline zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab-languages.xhtml]: Should do the same for zanata-war/src/main/webapp/WEB-INF/layout/version/settings-tab-languages.xhtml

Comment (aeng) [inline zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab-languages.xhtml]: @djansen-redhat Good one!

**Comment by djansen-redhat** [comment]
:white_check_mark: tested

**Comment by djansen-redhat** [comment]
Reviewed 3 of 3 files at r1, 2 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1248)*

**Pull request merged**